doc/api/cli.md

```diff
@@ -2294,6 +2294,25 @@ The following environment variables are set when running a script with `--run`:
 * `NODE_RUN_PACKAGE_JSON_PATH`: The path to the `package.json` that is being
   processed.
 
+### `--run-from=<path>`
```

Comment:
Should be documented in `node.1` as well?

Reply:
Yeah, that's a good point. @flakey5 ... can you update the `node.1` file also?